packages/common/src/lib/common/tools.ts

```diff
@@ -2080,7 +2077,7 @@ export async function getHostInfo(
 
     // Check if repositories exists
     const allRepos: Record<string, any> = {};
-    if (systemRepos && systemRepos.native && systemRepos.native.repositories) {
+    if (systemRepos?.native?.repositories && systemConfig) {
```

Comment:
```suggestion
    if (systemRepos?.native?.repositories && systemConfig?.common) {
```
or is that never undefined?

Reply:
According to types `system.config` always has a common property

Reply:
According to types yes, always defined